.travis.yml

```diff
@@ -7,10 +7,12 @@ addons:
 
 env:
   - DJANGO_SETTINGS_MODULE=settings
+sudo: required
```

Comment:
@moeez96 Why do we need root privilege here?

Reply:
Removed